frontend/src/metabase/services.js

```diff
@@ -315,7 +315,7 @@ export const TimelineEventApi = {
 };
 
 export const MetabaseApi = {
-  db_list: GET("/api/database", res => res["data"]),
+  db_list: GET("/api/database", ({ body }) => body["data"]),
```

Comment:
This file contains the new changes, because `master` doesn't have this line anymore. The new API expect that every thing would be passed in an object rather than passing `body` as the first argument.